app/frontend/src/test/utils.ts

```diff
@@ -3,7 +3,7 @@ import { createAction } from "@reduxjs/toolkit";
 export function addDefaultUser() {
   window.localStorage.setItem("auth.authenticated", JSON.stringify(true));
   window.localStorage.setItem("auth.jailed", JSON.stringify(false));
-  window.localStorage.setItem("auth.user", JSON.stringify(defaultUser));
+  window.localStorage.setItem("auth.userId", JSON.stringify(defaultUser));
```

Comment:
I think that might be the culprit of the function being called with an object! That should be setting to an ID!

Reply:
Good spot! I will fix it tomorrow